spec/features/index_organisation_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.feature "Listing Organisations" do
```

Comment:
Can we please keep the file name in sync with the feature? So it should be `listing_organisations_spec.rb`, I think that's more feature-like description.